pf9/support/generate_bundle.py

```diff
@@ -0,0 +1,199 @@
+import os
+import sys
+import click
+import getpass
+import shlex
+import subprocess
+import requests
+import json
+from fabric import Connection, Config
+import paramiko.ssh_exception
+import socket
+import invoke.exceptions
+import time
+from progress.bar import Bar
+from pf9.exceptions import DUCommFailure, CLIException, UserAuthFailure
+from pf9.modules.express import Get
+from pf9.modules.util import Utils, Logger, Pf9ExpVersion
+
+class Log_Bundle:
+
+    def __init__(self,error_msg="none"):
+        self.error_msg = error_msg
+
+    def get_uuid_from_resmgr(self,du_url,host_ip,headers):
+    #Based on the host_ip this function would pull the host UUID from resmgr.
+        try:
```

Comment:
In this method, you could ideally put all the logic in one big try... except block and then do the exception handling in just one place.

Reply:
Done.